.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,28 @@
+name: "Chromatic Deployment"
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+jobs:
+  test:
+    # Operating System
+    runs-on: ubuntu-latest
+    # Job steps
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Install dependencies
+        run: npm ci
+        #👇 Adds Chromatic as a step in the workflow
+      - uses: chromaui/action@v1
+        # Options required for Chromatic's GitHub Action
+        with:
+          #👇 Chromatic projectToken, see https://storybook.js.org/tutorials/intro-to-storybook/react/en/deploy/ to obtain it
+          projectToken: 65b519905a4d
```

Comment:
Leaving the chromatic project token as it is here because otherwise it wouldn't be possible to publish storybooks for external forks PRs. [More info about it](https://www.chromatic.com/docs/github-actions#run-chromatic-on-external-forks-of-open-source-projects).
Btw `app-services-ui-components` [does the same](https://github.com/redhat-developer/app-services-ui-components/blob/5a321948228d61253019065c0c8eae042bc54f36/.github/workflows/reusable-build.yml#L95).